action.yml

```diff
@@ -9,7 +9,7 @@ runs:
             chmod +x shuttle-darwin-amd64
             mkdir -p ~/bin
             mv shuttle-darwin-amd64 ~/bin/shuttle
-            shuttle --help
```

Comment:
you probably still want shuttle --help after you add it to the path, to make sure it is still reachable.

Reply:
I fixed this in the add m1 support pr instead.